### PR TITLE
fixes singup page overflow issue

### PR DESCRIPTION
**Pull request by love-sonkar**
fixes singup page profile picture is overflowing this is fixed in #1268 

ggg

gga

### CODE REVIEW

File: src/pages/Signup/index.css [hunk 1]
```diff
@@ -15,6 +15,7 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  margin-top:100px;
```
Comment (narayan954) [inline src/pages/Signup/index.css:18]: is this class even in use?

**Review by narayan954 — COMMENTED** [review]
className is not in use, so this property wont even work. Needs fix.

**Comment by narayan954** [comment]
I've added a fix, kindly verify it on your local by pulling it.
LMK if it's okay now?

**Review by love-sonkar — COMMENTED** [review]
Looks good 👍

**Review by narayan954 — APPROVED** [review]
lgtm

**Comment by narayan954** [comment]
Thanks for your contribution!

**Pull request merged**